packages/jupyterlab-preview/src/preview.tsx

```diff
@@ -98,14 +98,18 @@ export class VoilaPreview extends DocumentWidget<IFrame, INotebookModel> {
       icon: refreshIcon,
       tooltip: 'Reload Preview',
       onClick: () => {
-        this.reload();
+        context.save().then(() => {
```

Comment:
Minor styling note, we could also use `async` / `await` here for readability:

```ts
onClick: async () => {
  await context.save();
  this.reload();
}
```

Reply:
👍